prboom2/src/am_map.c

```diff
@@ -616,7 +617,7 @@ void AM_SetPosition(void)
     f_x = f_y = 0;
     f_w = SCREENWIDTH;
 
-    if (automap_overlay)
+    if (automap_overlay>0)
```

Comment:
Sorry to be picky, but I think we should try to use a uniform style for the new code: `if (automap_overlay > 0)` with spaces.

The original PrBoom+ was badly formatted, but all the new code from Kraflab has the same style (files in the dsda folder).

Reply:
Easy enough to fix 👍

Reply:
Should be resolved in latest commit